grails-app/migrations/0.9.x/changelog-2024-02-27-1630-add-tiered-pricing-to-product-supplier.groovy

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog = {
+    changeSet(author: "openboxes (generated)", id: "1709054923255-0") {
+        preConditions(onError: "HALT", onFail: "MARK_RAN", onSqlOutput: "IGNORE") {
+            not {
+                columnExists(columnName: "tiered_pricing", tableName: "product_supplier")
+            }
+        }
+
+        addColumn(tableName: "product_supplier") {
+            column(name: "tiered_pricing", type: "bit") {
+                constraints(nullable: "false")
```

Comment:
should it be `nullable: false` if it's considered as an optional field?

Reply:
I mean - what with already existing product sources? If we don't want to allow null, shouldn't we add a default bool value in migration?

Reply:
Existing ones got `false` by default.